demo/full/bxf-parser2/src/parser/schedule/schedule_elements/non_primary/overlay.js

```diff
@@ -0,0 +1,46 @@
+import reduceChildren from "../../../reduce_children";
+import { parsedTimeCode } from "../../../../utils/timecode";
+import parseTransition from "../../../node_commons/transition";
+import { parseStartDateTime } from "../../../node_commons/time";
+
+/**
+ * Parse logo event data
+ * @param {Element} node
+ */
+export default function parseOverlay(node, _name){
+  return reduceChildren(node, (res, name, node) => {
+    switch (name.toLowerCase()) {
+      case "ns1:eventtitle":
+        res.title = node.textContent;
+        break;
+      case "ns1:nonprimaryevent":
+        res.offset = reduceChildren(node, (res, name, node) => {
+          switch (name.toLowerCase()) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: "switch" statements should have at least 3 "case" clauses ([javascript:S1301](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS1301))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=366&issues=AWbt7Z3xW3Y69bz5aGXH&open=AWbt7Z3xW3Y69bz5aGXH)

[ISSUE_KEY]: AWbt7Z3xW3Y69bz5aGXH